guides/common/modules/con_configuring-virt-who-for-provider.adoc

```diff
@@ -0,0 +1,19 @@
+
+ifeval::["{context}" == "kubevirt-virt-who"]
+
+endif::[]
+
+[id="deploying-virt-who-configuration_{context}"]
+= Configuring virt-who for {provider}
+
+You create a virt-who configuration for {provider} by using the {ProjectWebUI} or the CLI on the {ProjectServer}.
```

Comment:
```suggestion
You create a virt-who configuration for {provider} by using the {ProjectWebUI} or the CLI on {ProjectServer}.
```

suggestion (non-blocking): I think we usually say "on {ProjectServer}" or "on your {ProjectServer}".